### PR TITLE
Increase character limit for Instructor feedback on ORAs

### PR DESCRIPTION
**Pull request by nsprenkle**
**TL;DR -** Increase character limit for Instructor feedback on ORAs.

JIRA: [EDUCATOR-5628](https://openedx.atlassian.net/browse/EDUCATOR-5628)

**What changed?**

- Increase the frontend character limit in feedback templates from 1k to 10k.

**Note:** the UI looks kind of gross and limits the horizontal space this feedback can take up. I wrote up a separate ticket ([EDUCATOR-5805](https://openedx.atlassian.net/browse/EDUCATOR-5805)) to change it.

**Developer Checklist**

- [x] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [x] Translations and JS/SASS compiled
- [x] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Testing Instructions**

1. Install and run migrations
2. In both a staff-graded ORA and peer-graded ORA, submit an ORA response as a student.
3. Provide assessments (as staff or peer) with between 1k and 10k characters to complete the ORA grading workflow.
4. View the "My Grade" section as a student, verify that the feedback sections are not clipped.
5. Attempt to provide an assessment with > 10k characters. Frontend should limit input to 10k characters (browser dependent). I've attached [a text file with just over 10k characters](https://github.com/edx/edx-ora2/files/6517105/large-feedback.txt) for easy testing. The trailing `THIS_SHOULD_BE_TRIMMED` should not appear.

Note: it is possible to bypass this on the frontend but the database can support much longer inputs.

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [x] I've done a visual code review
- [x] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

**Review by mattcarter — APPROVED** [review]
ten thousand char, check. Approved.

**Review by leangseu-edx — APPROVED** [review]
LGTM

**Review by jnlapierre — APPROVED** [review]
LGTM
